pkg/csi/service/wcpguest/controller.go

```diff
@@ -186,7 +186,7 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 	ctx = logger.NewContextWithLogger(ctx)
 	log := logger.GetLogger(ctx)
 	log.Infof("CreateVolume: called with args %+v", *req)
-	err := validateGuestClusterCreateVolumeRequest(ctx, req)
+	err := validateGuestClusterCreateVolumeRequest(ctx, req, commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.FileVolume))
```

Comment:
I think this is subjective. I would suggest to invoke IssFSSEnabled inside `validateGuestClusterCreateVolumeRequest` method. This way the validate method signature need not change every time we add a new feature.

Reply:
Makes sense, done